aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md

```diff
@@ -23,7 +23,7 @@ The following steps constitute the OBO process and the client credential process
 We will prepare two applications to demonstrate the dependent calls of resources.
 
 
-- Configure `Web API B` Resource Server: [azure-spring-boot-sample-active-directory-resource-server]
+- Configure `Web API B` Resource Server: [spring-cloud-azure-starter-active-directory-resource-server]
```

Comment:
the sample is not called `spring-cloud-azure-starter-active-directory-resource-server` and we don't have such library, should we improve these namings？